docs: Mention #argo-wf-sustainability in sustainability efforts (#12471)
argoproj/argo-workflows: community/sustainability_effort.md

Note that the hours per week listed above is an average over time; it's fine to have weeks of no activity so long as the average meets expectations.

> Participants should join the [`#argo-wf-contributors`](https://cloud-native.slack.com/archives/C0510EUH90V) Slack channel and ask to be invited to the private `#argo-wf-sustainability` channel.

Reviewer: ```suggestion
Participants should join the [`#argo-wf-contributors`](https://cloud-native.slack.com/archives/C0510EUH90V) Slack channel. They should also be invited to the private `#argo-wf-sustainability` channel.
```

Reviewer: what do you think of something like that?

Reviewer: I feel like if I saw this and somebody forgot to invite me, I would ask ?
